src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/gencode/GencodeFuncotationBuilder.java

```diff
@@ -163,7 +164,7 @@ public GencodeFuncotationBuilder setAnnotationTranscript(final String transcript
      * @param transcriptPos The position (1-based, inclusive) relative to the start of the transcript of a the variant in the {@link GencodeFuncotation}.
      * @return {@code this} {@link GencodeFuncotationBuilder}
      */
-    public GencodeFuncotationBuilder setTranscriptPos(final int transcriptPos) {
+    public GencodeFuncotationBuilder setTranscriptPos(final Integer transcriptPos) {
```

Comment:
this became an `Integer` to support the case where there's no transcript?

Reply:
Correct.